channels/ir.m3u

```diff
@@ -127,3 +127,19 @@ https://unirtmp.tulix.tv/tintv/tintv/chunklist.m3u8
 https://unirtmp.tulix.tv/tintv/tintv/playlist.m3u8
 #EXTINF:-1 tvg-id="YourTimeTV.ir" tvg-name="YourTime TV" tvg-country="IR" tvg-language="Persian" tvg-logo="https://yourtime.tv/img/utv-logo.png" group-title="Entertainment",YourTime TV
 https://hls.yourtime.live/hls/stream.m3u8
+#EXTINF:-1 tvg-id="Manoto.ir" tvg-name="Manoto" tvg-country="IR" tvg-language="Persian" tvg-logo="https://dwphh95xlnw84.cloudfront.net/images/static/placeholder/manotologo.png" group-title="Entertainment",Manoto TV
+https://d2rwmwucnr0d10.cloudfront.net/live.m3u8
+#EXTINF:-1 tvg-name="PMC" tvg-language="Farsi" tvg-country="IR" tvg-id="PMC" tvg-logo="https://upload.wikimedia.org/wikipedia/en/1/1f/Persian_Media_Corporation.jpg" group-title="Music",PMC (Music) (IR) (Farsi)
+https://hls.pmchd.live/hls/stream.m3u8
+#EXTINF:-1 tvg-name="VOA" tvg-language="Farsi" tvg-country="IR" tvg-id="VOA Farsi" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwngz4fqXSOIUrv1gWW3cZc-V-Zy7GfDJGvclL3j4Jno=s900-c-k-c0x00ffffff-no-rj" group-title="Music",VOA Farsi
+https://voa-lh.akamaihd.net/i/voapnn_7@72817/master.m3u8
+#EXTINF:-1 tvg-name="BBCPersian" tvg-language="Farsi" tvg-country="IR" tvg-id="BBC Persian" tvg-logo="https://news.files.bbci.co.uk/ws/img/logos/og/persian.png" group-title="Food",BBC Persian
+https://vs-hls-ww.live.cf.md.bbci.co.uk/pool_902/live/nonuk/bbc_persian_tv/bbc_persian_tv.isml/bbc_persian_tv-pa4=128000-video=2812032.m3u8
+#EXTINF:-1 tvg-name="GEMfood" tvg-language="Farsi" tvg-country="IR" tvg-id="GEM food" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_food.png?14" group-title="Music",GEM Food
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_food/playlist_960x540_2000k.m3u8
```

Comment:
Broken Link

Reply:
I check the BBC it is working only the Gem does not work which i guess it is from their channel they will fix it soon since it does not show on their own offical website too

https://www.gemonline.tv/